scripts/cddl_gen.py

```diff
@@ -1959,10 +1973,10 @@ def xcoders(self):
             for xcoder in self.my_types[self.value].xcoders():
                 yield xcoder
         if self.repeated_single_func_impl_condition():
-            yield (self.repeated_xcode(), self.repeated_xcode_func_name(), self.repeated_type_name())
+            yield xcoder_tuple(self.repeated_xcode(), self.repeated_xcode_func_name(), self.repeated_type_name())
```

Comment:
With my `typing.NamedTuple` suggestion above, this could yield XCoder classes instead of what looks like whatever the `xcoder_tuple` function yields (though I know it's not a function, just looks so here).